tests/integration/test_project.py

```diff
@@ -191,3 +201,25 @@ def test_queue_mode(configured_project: Project):
     ) == configured_project.QueueMode.Dataset
     configured_project.update(queue_mode=configured_project.QueueMode.Batch)
     assert configured_project.queue_mode() == configured_project.QueueMode.Batch
+
+
+def test_media_type(client, configured_project: Project, rand_gen):
+    # Existing project
+    assert configured_project.media_type is None or isinstance(
+        configured_project.media_type, Project.MediaType)
+
+    # No media_type
+    project = client.create_project(name=rand_gen(str))
```

Comment:
do we need to create a new project to test this? or can we just use the configured project for this?

Reply:
Test modified as suggested.